core/src/executor/context/blend_context.rs

```diff
@@ -25,15 +31,21 @@ impl<'a> BlendContext<'a> {
             next,
         }
     }
+    // pub fn get_table_alias(&'a self) -> &'a str {
+    //     self.table_alias
+    // }
 
     pub fn get_value(&'a self, target: &str) -> Option<&'a Value> {
         let get_value = || {
             self.columns
                 .iter()
                 .position(|column| column == target)
                 .map(|index| match &self.row {
-                    Some(row) => row.get_value(index),
-                    None => Some(&Value::Null),
+                    BlendContextRow::Single(Some(row)) => row.get_value(index),
+                    BlendContextRow::Shared(Some(row)) => row.get_value(index),
+                    BlendContextRow::Single(None) | BlendContextRow::Shared(None) => {
+                        Some(&Value::Null)
+                    }
```

Comment:
Then, we don't need to consider `BlendContext::Shared(None)` case so we can simplify like..

```rust
BlendContextRow::Shared(row) => row.get_value(index),
BlendContextRow::Single(Some(row)) => row.get_value(index),
BlendContextRow::Single(None) => Some(&Value::Null),
```